tests/certification/bindings/azure/blobstorage/blobstorage_test.go

```diff
@@ -558,16 +558,16 @@ func TestBlobStorage(t *testing.T) {
 
 		cred, _ := azblob.NewSharedKeyCredential(os.Getenv("AzureBlobStorageAccount"), os.Getenv("AzureBlobStorageAccessKey"))
 		service, _ := azblob.NewServiceClientWithSharedKey(fmt.Sprintf("https://%s.blob.core.windows.net/", os.Getenv("AzureBlobStorageAccount")), cred, nil)
-		containerClient := service.NewContainerClient(os.Getenv("AzureBlobStorageContainer"))
+		containerClient, _ := service.NewContainerClient(os.Getenv("AzureBlobStorageContainer"))
```

Comment:
Should we handle the error and fail the test?

Reply:
Normally I'd say we should handle it but it really doesn't matter here which is why I didn't :)